docs/guides/cd.md

```diff
@@ -1,64 +1,64 @@
 # Continuous Deployment of your Functions
 
-To continously deploy your functions on TriggerMesh you can register a source code repository which contains your function code and a manifest (`serverless.yaml`).
 
-The **_Repositories_** section in the console contains a view to register any repository, from previously linked accounts, for continuous deployment.
 
+The Triggermesh console provides an interface called [Repositories.](https://cloud.triggermesh.io/function-triggers) Here the user is able to select a codebase, from a linked repository provider, to be continuously deployed. This codebase **must** contain a deployment manifest[^1] . 
 
-## Register the repository
 
+## Register the repository
 
-**Begin by forking the [sample repository](https://github.com/sebgoa/tmserverless) which contains a function and an example `serverless.yaml` manifest.**
+###### The following example uses this [sample repository.](https://github.com/sebgoa/tmserverless) To follow along begin by forking the example to your repository provider. 
 
-Select the _Repositories_ section from the Navigation menu and then _CREATE NEW_ 
+1. Select the _Repositories_ section from the Navigation menu and then _CREATE NEW_ 
 
 ![](../images/tmRepoView.png)
 
-Select a source control provider. Upon submission of this form you will be asked to authenticate with the providor chosen in the previous step.
+2. Select a source control provider. Upon submission of this form you will be asked to authenticate with the chosen provider.
 
 ![](../images/reopwiz1.png)
 
 
-After authentication, Select the  [sample repository](https://github.com/sebgoa/tmserverless) that was forked in the beginning from the drop down list.
+3. Select the  [sample repository](https://github.com/sebgoa/tmserverless) from the drop down list.
 
 ![](../images/repowiz2.png)
 
-Here you can specify the location of the `serverless.yaml` manifest as well as select specific _Branch_ and _Tag_ name's from the repository. For this example we can 
+4. Here you can specify the location of the `serverless.yaml` manifest as well as select specific _Branch_ and _Tag_ name's from the repository. 
 
 ![](../images/repowiz3.png)
 
-Once registration is complete the _Repositories_ view will contain your function project. You can disable the registration at any time by selecting the project and clicking on the _Disable Selected_ button.
+5. Once registration is complete the _Repositories_ view will contain your function project. You can disable the registration at any time by selecting the project and clicking on the _Trash Can_ icon located next to _REFRESH_
 
 ![](../images/repowiz4.png)
 
 ## Viewing The Deployment
 
-After every sucessful registration of a new repository a _Task_ will be created for it.
+###### After every sucessful registration of a new repository a _Task_ will be created for it.
```

Comment:
Why the sublevel here?  Perhaps something to emphasize the text?

Reply:
This one I agree should be put back to bold.